package.json

```diff
@@ -15,7 +15,8 @@
     "htmlparser2": "3.8.3",
     "cssstyle": "0.2.29",
     "webpack": "1.12.2",
-    "webpack-dev-server": "1.12.1"
+    "webpack-dev-server": "1.12.1",
+    "i18next": "3.0.0"
```

Comment:
@timabbott, we will have to if we are using the package from `node_modules` directory. We can add these libraries directly to `static/third` and then include in `JS_SPEC` but this way just doing `require('i18next')` includes these packages in the bundle.